mobile_verifier/src/cli/server.rs

```diff
@@ -101,6 +101,8 @@ impl Cmd {
             settings.usa_and_mexico_fencing_resolution()?,
         )?;
 
+        let (new_coverage_obj_signal_tx, new_coverage_obj_signal_rx) = channel(1);
```

Comment:
This is good. I think it does a good job of clarifying which daemon is responsible for what, and why they care about each other. :+1: